tpv/commands/test/mock_galaxy.py

```diff
@@ -76,10 +78,11 @@ def __init__(self, job_conf=None, create_model=False):
 
 
 class User:
-    def __init__(self, username, email, roles=[]):
+    def __init__(self, username, email, roles=[], id=None):
         self.username = username
         self.email = email
         self.roles = [Role(name) for name in roles]
+        self.id = id or random.randint(1, 1000000)
```

Comment:
Instead of a random integer, if the id is not defined maybe it would be better to compute it from the username and/or email with something like a hash function? Just for reproducibility.

Reply:
good thinking